yoojin_lee/최댓값-만들기(1).js

```diff
@@ -0,0 +1,5 @@
+function solution(numbers) {
+    // 정렬 후 곱
+    numbers = numbers.sort((a,b) => b-a);
+    return (numbers[0] * numbers[1]);
+}
```

Comment:
우와 새로운 풀이법을 익혀갑니다~!!👍
```suggestion
function solution(numbers) {
    // 정렬 후 곱
    numbers = numbers.sort((a,b) => b-a);
    return (numbers[0] * numbers[1]);
}
```

Reply:
감쟈합니다~!!🤭